docs/src/css/style.css

```diff
@@ -241,6 +241,12 @@ a.scroll-link:hover {
   box-sizing: border-box;
   padding: 2.5rem;
 }
+
+.overflow{
+   padding:10px;
+   display:block;
+   }
+  
 @media only screen and (max-width: 1024px) {
```

Comment:
Bulma has its own [breakpoints mixins](https://bulma.io/documentation/overview/responsiveness/#breakpoints) to handle screen sizes, there's no need to add new hardcoded screen sizes

Reply:
@hugosolar they are from my previous commit, the latest commit doesn't use the hardcoded sizes.

Reply:
I think I'm watching the last changes on the files. please double check if you commit your latest changes.

Reply:
yes I did commit it, but didn't remove the CSS changes, I didn't use them in the html code tho. I have used the bulma breakpoints only, no hardcoding this time.

Reply:
commit ada8a2fd47e9225a8109c51f43f995d7b0b42f5e please refer to this commit's HTML file

Reply:
Yes. that commit only changes HTML, although I can't see the use of bulma's responsive columns you just change the column display for all sizes